### PR TITLE
Isolate Assignment Enum

### PR DESCRIPTION
**Pull request by xiaobill8**

### CODE REVIEW

File: src/main/java/seedu/address/model/task/enums/Assignment.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * or the user has assigned to someone else.
+ */
+public enum Assignment {
+    FROM, TO;
+
+    @Override
+    public String toString() {
+        switch (this) {
+        case FROM:
+            return "Assigned from";
```
Comment (dexter-sim) [inline src/main/java/seedu/address/model/task/enums/Assignment.java:14]: I think the correct grammar should be `"Assigned by"`.

Comment (xiaobill8) [reply src/main/java/seedu/address/model/task/enums/Assignment.java]: Good grammar!

**Review by seetohjinwei — APPROVED** [review]
+1

Looks good to me! Much clearer than before!

**Review by dexter-sim — APPROVED** [review]
lgtm

**Pull request merged**